src/nav/full-stack-observability.yml

```diff
@@ -2190,6 +2190,8 @@ pages:
                 path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/vector-output-sink-log-forwarding
               - title: Stream logs from Heroku
                 path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/heroku-log-forwarding
+              - title: Azure logs ARM template
+                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/azure-log-forwarding
```

Comment:
Hi reviewer! This is the only real change on this file, the others are trimmed whitespaces from the end of the lines, I can put them back if you want.

Reply:
No worries, I noticed that the diff are the spaces. It's not a problem 🙌🏻